### PR TITLE
The code to process the geography lookup file expects lowercase file suffixes cd, nm and nmw

### PR DESCRIPTION
**Pull request by phynes-sensiblecode**
This is in line with the delivered lookup file.

Updated to 1.2.epsilon

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Release Notes
 =============
 
+1.2.epsilon
+-----------
+- The code to process the geography lookup file expects lowercase file prefixes `cd`, `nm` and `nmw`.
```
Comment (SpencerHedger) [inline RELEASE_NOTES.md:6]: Do you mean suffix?

**Review by SpencerHedger — CHANGES_REQUESTED** [review]
Just a typo around prefix/suffix

File: RELEASE_NOTES.md (hunk 1 above)
Comment (phynes-sensiblecode) [reply RELEASE_NOTES.md:6]: Yes

**Review by SpencerHedger — APPROVED** [review]
Looks good, made a further commit for you to fix another prefix/suffix typo, take a look and merge if happy.

**Pull request merged**